modules/ocf_backups/files/upload-to-box

```diff
@@ -164,6 +105,36 @@ def get_access_token(creds):
         return response['access_token']
 
 
+def upload_to_box(creds, quiet):
+    """Uploads the archive to box at the configured location"""
+    lftp_cmd = '''
+        set ftps:initial-prot ""
+        set ftp:ssl-force true
+        set ftp:ssl-protect-data true
+        open {box_ftp}
+        user {email} {passwd}
+        mirror -R --parallel={concurr} {archive} {folder}
```

Comment:
I know @jvperrin did some benchmarking. Did you try seeing if the box.com server supports `zlib` compression with:

```
set ftp:use-mode-z true
set ftp:mode-z-level 6
```

Since you are using mirror, you may also want to consider: 
```
set mirror:parallel-directories true
```

This will do directories in parallel, if possible, since you already pass the `--parallel` flag.

Reply:
since the files we are uploading are encrypted, compression will just burn CPU as pseudorandom data is nearly incompressible

Reply:
ah, good point.

Reply:
Don't think `parallel-directories` will do anything either since this just one directory with a bunch of flat files in it.